automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/FireCloudClient.scala

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.model.{Multipart, _}
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, _}
+import akka.util.ByteString
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import org.broadinstitute.dsde.firecloud.config.AuthToken
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor, Future}
+
+class FireCloudClient {
```

Comment:
Actually, I'm not sure why the things below this (`Orchestration`, `Rawls`, `Thurloe`) are traits with companion objects instead of just straight-up objects. @rtitle can you think of any reason not to make `FireCloudClient` a trait and the rest objects?